infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
+		Labels:  []string{"instance"},
+		Comment: "The number of leaders being written on each TiKV instance",
+	},
+	"tikv_approximate_region_size": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_raftstore_region_size_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,instance))`,
+		Labels:   []string{"instance"},
+		Quantile: 0.99,
+		Comment:  "The approximate Region size",
+	},
+	"tikv_approximate_avg_region_size": {
+		PromQL:  `sum(rate(tikv_raftstore_region_size_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) / sum(rate(tikv_raftstore_region_size_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) `,
+		Labels:  []string{"instance"},
+		Comment: "The approximate avg Region size",
+	},
+	"tikv_approximate_region_size_histogram": {
+		PromQL: `sum(rate(tikv_raftstore_region_size_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_region_average_written_bytes": {
+		PromQL:  `sum(rate(tikv_region_written_bytes_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance) / sum(rate(tikv_region_written_bytes_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The average rate of writing bytes to Regions per TiKV instance",
+	},
+	"tikv_region_written_bytes": {
+		PromQL: `sum(rate(tikv_region_written_bytes_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_region_average_written_keys": {
+		PromQL:  `sum(rate(tikv_region_written_keys_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance) / sum(rate(tikv_region_written_keys_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The average rate of written keys to Regions per TiKV instance",
+	},
+	"tikv_region_written_keys": {
+		PromQL: `sum(rate(tikv_region_written_keys_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_request_batch_avg": {
+		PromQL:  `sum(rate(tikv_server_request_batch_ratio_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_server_request_batch_ratio_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The ratio of request batch output to input per TiKV instance",
+	},
+	"tikv_request_batch_ratio": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_server_request_batch_ratio_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,type,instance))`,
+		Labels:   []string{"instance", "type"},
+		Quantile: 0.99,
+		Comment:  "The ratio of request batch output to input per TiKV instance",
+	},
+	"tikv_request_batch_size_avg": {
+		PromQL:  `sum(rate(tikv_server_request_batch_size_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_server_request_batch_size_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The avg size of requests into request batch per TiKV instance",
```

Comment:
```suggestion
		Comment: "The avg size of requests in request batch per TiKV instance",
```

Reply:
`into` is ok here, it indicates `divide`.